packages/apollo-server-express/src/__tests__/expressApollo.test.ts

```diff
@@ -22,6 +23,12 @@ describe('expressApollo', () => {
       'ApolloServer requires options.',
     );
   });
+
+  it('throws error if called with more than one argument', function() {
+    expect(() => graphqlExpress({ schema: Schema }, 1)).toThrow(
+      'Apollo Server expects exactly one argument, got 2'
```

Comment:
Likewise, Prettier wants a trailing comma on the end of this line.  This is to try to facilitate the _potential_ addition of another argument to `toThrow` in the future, even if you haven't done that today.